packages/plugins/plugin-inbox/src/components/Mailbox/mailbox.css

```diff
@@ -10,7 +10,7 @@
   & > .dx-grid__cell__content,
   &.message__card {
     /** TODO(burdon): Reconcile with styles.ts (mailboxGrid) */
-    @apply grid grid-cols-[3rem_1fr] gap-1 border-0;
+    @apply grid grid-cols-[3rem_1fr] gap-1 p-1 border-0;
```

Comment:
Please be sure you’ve confirmed this is the desired padding in all the places it applies, including both regular `Mailbox` as well as a `ProjectContainer` with messages.

Reply:
now removed